data/virt_autotest/guest_params_xml_files/sles_15_sp7_64_kvm_hvm_uefi_non_secure_boot_x86_64.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>sles</guest_os_name>
+    <guest_version>15-sp7</guest_version>
+    <guest_version_major>15</guest_version_major>
+    <guest_version_minor>7</guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <guest_build/>
+    <host_hypervisor_uri/>
+    <host_virt_type>kvm</host_virt_type>
+    <guest_virt_type>hvm</guest_virt_type>
+    <guest_machine_type/>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name/>
+    <guest_domain_name/>
+    <guest_memory>2048,maxmemory=4096</guest_memory>
+    <guest_vcpus>2,maxvcpus=4</guest_vcpus>
+    <guest_cpumodel/>
+    <guest_metadata/>
+    <guest_xpath>./os/boot[@dev='hd']/@order=1#./os/boot[@dev='network']/@order=2</guest_xpath>
+    <guest_installation_automation_method>autoyast</guest_installation_automation_method>
+    <guest_installation_automation_platform/>
+    <guest_installation_automation_file>sles_15_plus_kvm_hvm_guest_graphical_x86_64.xml</guest_installation_automation_file>
+    <guest_installation_method>location</guest_installation_method>
+    <guest_installation_extra_args>sshd=1#sshpassword=ROOTPASSWORD#console=ttyS0,115200n8#textmode=1</guest_installation_extra_args>
+    <guest_installation_wait/>
+    <guest_installation_method_others/>
+    <guest_installation_media>http://openqa.suse.de/assets/repo/SLE-15-SP7-Full-x86_64-Build12345-Media1/</guest_installation_media>
+    <guest_boot_settings>loader=/usr/share/qemu/ovmf-x86_64-code.bin,loader.readonly=yes,loader.type=pflash</guest_boot_settings>
+    <guest_secure_boot>false</guest_secure_boot>
```

Comment:
No secure boot vm is needed? Seems there is mismatch with PR description.

Reply:
Typo in description, it should be non-secure boot :rofl:  , just corrected